browser/css/jssidebar.css

```diff
@@ -395,6 +395,8 @@ button#button2.ui-pushbutton.jsdialog.sidebar {
 	/* Move this whole block away and fix in the main control */
 	display: flex;
 	align-items: center;
+	width: 200px;
+	word-break: break-all;
```

Comment:
# Breaking

Best to use `break-word` instead otherwise the the text will break everywhere making it hard to read

| break-all | break-word |
| ------------- | -----------------|
| `https://en.wikipedia.org/wiki/Sai` | `https://en.wikipedia.org` | 
| `ling_yacht` | `/wiki/Sailing_yacht` |

Reply:
i used break-all cause it looked even on both the browsers chrome and firefox , but you're right thank you.